query-engine/driver-adapters/js/connector-test-kit-executor/src/index.ts

```diff
@@ -204,10 +203,30 @@ function respondOk(requestId: number, payload: unknown) {
     console.log(JSON.stringify(msg))
 }
 
-async function initQe(url: string, prismaSchema: string, logCallback: qe.QueryLogCallback): Promise<[engines.QueryEngineInstance, ErrorCapturingDriverAdapter]> {
+/**
+ * Extract the schema from the url params
+ * @param url the prisma connection URL
+ */
+function extractSchema(url: string): string | null {
+    const schemaRegex = /[?&]schema=([^&]+)/;
+    const match = url.match(schemaRegex);
+    if (match && match[1]) {
+        return match[1];
+    } else {
+        return null;
+    }
+}
```

Comment:
It's for tests so we don't care that much but I think you could've used the [`URL`](https://nodejs.org/api/url.html) module for that instead. eg:

```ts
new URL(url).searchParams.get('schema')
```